nixodmlconverter/convert.py

```diff
@@ -197,12 +197,15 @@ def odml_to_nix_property(odmlprop, nixsec):
         if nixv is not None:
             nixvalues.append(nixv)
 
-    if not nixvalues:
-        INFO["skipped empty properties"] += 1
-        return
+    #if not nixvalues:
+    #    INFO["skipped empty properties"] += 1
+    #    return
 
     # We need to get the appropriate NIX DataType for the current odML values
-    dtype = nix.DataType.get_dtype(nixvalues[0])
+    if nixvalues:
+        dtype = nix.DataType.get_dtype(nixvalues[0])
+    else:
+        dtype = nix.DataType.get_dtype("")
```

Comment:
Would it be simpler to set the dtype directly to `nix.DataType.String` and avoid needing to run the detection function?  It's not costly or anything, but perhaps more readable and explicit.